app/server/app/routes/help.js

```diff
@@ -0,0 +1,69 @@
+const express = require("express");
+const axios = require("axios").default;
+// ---
+const {
+  formioProjectUrl,
+  formioFormId,
+  formioHeaders,
+} = require("../config/formio");
+const { ensureAuthenticated } = require("../middleware");
+const logger = require("../utilities/logger");
+
+const log = logger.logger;
+
+const router = express.Router();
+
+// TODO: update authentication check to restrict these API calls to users in the correct EPA WAA groups
+router.use(ensureAuthenticated);
```

Comment:
@dpgalloway I'd imagine you'll want to create a similar `ensureAuthenticated` middleware, specifically for help desk users that also checks their EPA WAA groups. You can apply it to this entire file (as done here) as these routes are only called by help desk users.